src/utils/fees.ts

```diff
@@ -27,7 +27,7 @@ export const useFees = () => {
       if (!balance) return new Balance<DataCredits>(0, CurrencyType.dataCredit)
 
       const prices = [currentOraclePrice, ...predictedOraclePrices]
```

Comment:
If `currentOraclePrice` is undefined, would this fail?

Reply:
In the old app it didn't default to 0, I think lodash just ignores the undefined ones. If you default it to 0, then the min would always be 0 and the fee would be incorrect.

Reply:
ok, that works.